pkg/Library/UX/UXNodeTypes.js

```diff
@@ -28,7 +28,7 @@ export const UXNodeTypes = {
     category,
     description: 'Renders a message in a temporary popup',
     types: {
-      UXSnackBar$open: 'Nonce',
+      UXSnackBar$open: 'Nonce|String',
```

Comment:
I'm just curious about this?

Reply:
here I want to be able to connect the `message` and the `open` to the same output, so that the SnackBar appear every time a new message is emitted. 
Similar change in the OpenAI nodes, so i can connect the `prompt` and `restart` inputs to the same string output, to trigger regeneration whenever the prompt changes...

PS: there is a flaw here, because the way I use SnackBar in Build, it is theoretically possible for the same text to be outputted twice in a raw, but represent 2 separate messages.. Actually this is an upstream problem, so irrelevant for the purpose of this PR (also our current message include the graph/node name, so should be unique)

Reply:
Thanks for the info!

Seems like we need 'versioned primitives' somehow. In the same way Image is {[url|canvas], version}.